DxDispatch/src/dxdispatch/Device.cpp

```diff
@@ -113,6 +114,7 @@ Device::Device(
         IID_GRAPHICS_PPV_ARGS(m_fence.ReleaseAndGetAddressOf())));
 
     D3D12_COMMAND_QUEUE_DESC queueDesc = {};
+    queueDesc.Flags = D3D12_COMMAND_QUEUE_FLAG_DISABLE_GPU_TIMEOUT;
```

Comment:
>queueDesc.Flags = D3D12_COMMAND_QUEUE_FLAG_DISABLE_GPU_TIMEOUT; [](http://example.com/codeflow?start=4&length=63)

should this be an commandline option? And have D3D12_COMMAND_QUEUE_FLAG as an arg to the device creation?

Reply:
Good idea!